src/net.h

```diff
@@ -75,7 +75,9 @@ static const int MAX_FEELER_CONNECTIONS = 1;
 /** -listen default */
 static const bool DEFAULT_LISTEN = true;
 /** The maximum number of peer connections to maintain. */
-static const unsigned int DEFAULT_MAX_PEER_CONNECTIONS = 125;
+static const unsigned int DEFAULT_MAX_PEER_CONNECTIONS{200};
+/** Percentage of inbound connection slots that tx-relaying peers can use */
+static const float FULL_RELAY_INBOUND_PCT{0.6};
```

Comment:
No idea why windows CI runs on your repo, but it's nice. It complains about the float - will change it to double later.

Reply:
actually changed it to int because it's a percentage.